db/schema.rb

```diff
@@ -16,10 +16,10 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
```

Comment:
I have opend another PR for tracking changes to schema.rb. Can you please remove these changes from this PR?